Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/Sharing/SpatialCoordinateSystemParticipant.cs

```diff
@@ -6,267 +6,98 @@
 using Microsoft.MixedReality.Toolkit.Utilities;
 using System;
 using System.IO;
+using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
 using UnityEngine;
+using UnityEngine.XR.WSA;
 
```

Comment:
Why is this needed/does it build for ios/android?

Reply:
This is used to determine if the device is tracking for the PeerDeviceHasTracking property. I've changed it so that this value can be platform-specific in SendCoordinateStateMessage. Might need help with getting this right for Android

Reply:
This was for tracking status, that's been changed to have a platform-specific switch.